docs/resources/call_control_application.md

```diff
@@ -0,0 +1,5 @@
+# telnyx_call_control_application Resource
+
+Provides a Telnyx Call Control Application resource. This resource allows you to create, read, update, and delete call control applications via the Telnyx API.
+
+<!-- More documentation will be added after implementation. -->
```

Comment:
I think these need to be gened with [this](https://github.com/petsinc/terraform-provider-telnyx/blob/main/provider/Makefile#L44-L45). I guess we should probably move all of this doc gen stuff into the root justfile and delete the makefiles as long as we've got everything we need out of them.